cmd/sourced/compose/workdir/handler.go

```diff
@@ -0,0 +1,191 @@
+package workdir
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+)
+
+type Handler struct {
```

Comment:
it's public struct, please doc

Reply:
I'm gonna take a look over all public things and document them 👍